m46x/StdDriver/drv_emac/synopGMAC_network_interface.h

```diff
@@ -0,0 +1,63 @@
+/* ===================================================================================
+ * Copyright (c) <2009> Synopsys, Inc.
+ *
```

Comment:
Apache-2.0 license  please

Reply:
This code was developed by Synopsys to be tested with Linux OS (Kernel Version 2.6.xy). And Synopsys BU confirmed it's fine to add extra license string as:
```
/* ===================================================================================
 * Copyright (c) <2009> Synopsys, Inc.
 * SPDX-License-Identifier: Apache-2.0
 *
 * Permission is hereby granted, free of charge, to any person obtaining a copy of

```


> Please sync SDK based on release, which help to avoid conflict of partial update of Stddriver and reset out of sync. I would suggest to drop this change set.

Rebase & fixed in commit d1b4db369496af3b9279b45fd2ce9fbe18596776